src/routes/view-content/bible-menu/BibleMenu.svelte

```diff
@@ -27,15 +28,11 @@
 </script>
 
 <div class="z-50 flex h-full w-full flex-col">
-    <div class="relative mb-6 flex h-[166px] w-full rounded-b-3xl bg-[#6172F3] ps-6 pt-12">
-        <div class="absolute bottom-0 left-0 w-full">
-            <img src="/menu-swish.png" alt="Menu Swish" class="h-auto w-full rounded-b-3xl" />
-        </div>
-        <div class="flex flex-col">
-            <h2 class="z-10 mb-2 text-2xl font-bold text-white">{$translate('page.bibleMenu.bible.value')}</h2>
-            <span class="z-10 text-sm text-white">{$translate('page.bibleMenu.selectBible.value')}</span>
-        </div>
-    </div>
+    <SwishHeader
+        bgcolor="bg-[#6172F3]"
+        title={$translate('page.bibleMenu.bible.value')}
+        subtitle={$translate('page.bibleMenu.selectBible.value')}
+    />
```

Comment:
Here and for the GuideMenu I swapped in this component. The component was already built out but only used in one place.